src/main/java/slimeknights/tconstruct/tables/client/inventory/module/SideButtonsWidget.java

```diff
@@ -12,33 +12,28 @@
 
 public class SideButtonsWidget<T extends Button> implements Widget, GuiEventListener {
 
-  protected final MultiModuleScreen<?> parent;
+  public static final int SPACING = 4;
```

Comment:
Is this used outside the file at all? I have a suspicion the intention was to allow changing the spacing on a per widget basis, so may be better to make it private/protected to make a refactor later lower risk

Reply:
Sounds reasonable.